Tools/autotest/sim_vehicle.py

```diff
@@ -1062,6 +1079,10 @@ def generate_frame_help():
                   help="be compatible with the way sim_vehicle.sh works; "
                   "make this the first option")
 
+parser.add_option("-P", "--param",
```

Comment:
This should probably look much like `--configure-define` so multiple can be added.